src/desktop/package.json

```diff
@@ -133,27 +132,27 @@
     "bugsnag-js": "^4.7.3",
     "bugsnag-react": "^1.1.1",
     "classnames": "^2.2.6",
-    "curl.lib.js": "^1.0.22",
     "electron-settings": "^3.2.0",
     "electron-updater": "^3.1.2",
     "entangled-node": "rihardsgravis/entangled-node",
+    "fsevents": "1.2.4",
     "i18next": "^11.9.0",
     "iota.lib.js": "^0.5.1",
     "kdbxweb": "^1.2.4",
-    "keytar": "^4.2.1",
+    "keytar": "^4.3.0",
     "machine-uuid-sync": "https://github.com/rajivshah3/machine-uuid-sync",
     "perfect-scrollbar": "^1.4.0",
     "qr.js": "^0.0.0",
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
-    "react-i18next": "^7.12.0",
+    "react-i18next": "^8.0.7",
     "react-lottie": "^1.2.3",
-    "react-markdown": "^3.6.0",
-    "react-qr-reader": "^2.1.0",
+    "react-markdown": "^4.0.2",
```

Comment:
`react-markdown` shouldn't be updated past 3.6.0 unless absolutely necessary. Later versions introduce a vulnerable dependency (https://app.snyk.io/test/npm/react-markdown/4.0.2)